tests/api/handlers/test_utils.py

```diff
@@ -0,0 +1,8 @@
+from fmn.api.handlers import utils
+from fmn.rules.requester import Requester
+
+
+def test_gen_requester():
```

Comment:
I don't think anyone uses the `gen_requester()` function anymore, it can be removed as well.

Reply:
Good catch!